docs/source/app-dev/bindings-js/index.rst

```diff
@@ -7,16 +7,50 @@ Node.js bindings
 .. toctree::
   :hidden:
 
-  getting-started
   tutorial
   daml-as-json
 
 The Node.js bindings let you access the Ledger API from Node.js code.
 
-The Installing page guides you through the process of :doc:`getting-started`.
+Getting started
+***************
 
-After that, a tutorial teaches you :doc:`how to build a simple application from scratch <tutorial>`.
+1. Set up NPM
+=============
 
-Finally, `the API reference documentation`_ and :doc:`an introduction to DAML as JSON <daml-as-json>` are also available.
+The Node.js bindings are published on NPM.
 
-.. _the API reference documentation: ./reference/index.html
+To set up NPM:
```

Comment:
Since you already mention that we are publishing on NPM, these setup instructions are no longer needed.

If by "published on NPM" you meant the public one, which would likely be implicitly assumed if you don't explicitly say "published on DA's public NPM repository on Bintray".